ophirofox/manifest.json

```
@@ -335,6 +335,10 @@
           "name": "Université d'Artois",
           "AUTH_URL": "http://ezproxy.univ-artois.fr/login?url=https://nouveau.europresse.com/access/ip/default.aspx?un=littoralT_1"
         },
+        {
+          "name": "Université d'Évry Val d'Essonne",
+          "AUTH_URL": "https://nouveau-europresse-com.ezproxy.universite-paris-saclay.fr/access/ip/default.aspx?un=U031535T_8"
```

Est-ce que vous êtes sûr pour l'URL ? Selon la logique des autres universités, est-ce que ce ne serait pas plutôt ?

```suggestion
          "AUTH_URL": "https://ezproxy.universite-paris-saclay.fr/login?url=http://nouveau.europresse.com/access/ip/default.aspx?un=U031535T_8"
```